### ISSUE TITLE
Dashboard: My Stories; A11y, Search Component cannot be focused

### ISSUE THREAD

**Issue by carloskelly13** [GoogleForCreators/web-stories-wp#4621]
## Bug Description

The search component is unable to be focused on the dashboard, my stories view.

## Additional Context

- Plugin Version:
- Operating System:
- Browser:

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

**Comment by csossi** [comment]
Verified in QA
